topomodelx/nn/simplicial/sccn.py

```diff
@@ -47,19 +49,12 @@ def forward(self, features, incidences, adjacencies):
 
         Parameters
         ----------
-        features: Dict[int, torch.Tensor],
-                length=max_rank+1,
-                shape=[n_rank_r_cells, channels]
+        features: Dict[int, torch.Tensor], length=max_rank+1, shape=[n_rank_r_cells, channels]
```

Comment:
`dict` in lowercase, please (same below) 🙂

Reply:
Tagging @jarpri.